presentation/src/main/java/com/android254/presentation/home/components/HomeSectionHeaderComponent.kt

```diff
@@ -101,14 +80,10 @@ fun HomeSectionHeaderComponent(
                         shape = RoundedCornerShape(14.dp)
                     )
             ) {
-                Text(
-                    text = stringResource(id = R.string.format_plus_label, sectionSize),
+                ChaiTextLabelMedium(
                     modifier = Modifier.align(Alignment.Center),
-                    style = TextStyle(
-                        color = MaterialTheme.chaiColorsPalette.linkTextColorPrimary,
-                        fontSize = 10.sp,
-                        fontFamily = MontserratRegular
-                    )
+                    bodyText = stringResource(id = R.string.format_plus_label, sectionSize),
```

Comment:
minor: we can have a `ChaiTextLabelMedium` parameter that takes in string resource Id so that we just pass `R.string.my_string`